setup.py

```diff
@@ -96,7 +96,7 @@ def build_extension(self, ext):
 
         cmake_args = [
             f"-DCMAKE_BUILD_TYPE={cfg}",
-            # f"-DCMAKE_PREFIX_PATH={torch.utils.cmake_prefix_path}",
+            f"-DCMAKE_PREFIX_PATH=/Users/nayef211/opt/miniconda3/envs/torchtext/lib/python3.9/site-packages/torch/share/cmake",
```

Comment:
you can also use `{torch.utils.cmake_prefix_path}`, similar to TorchAudio:

https://github.com/pytorch/audio/blob/4b47412e91c22d75a1d5f4ea5f71e0ceaa26efd7/tools/setup_helpers/extension.py#L93